setup.cfg

```diff
@@ -16,11 +16,15 @@ branch = True
 
 [coverage:report]
 precision = 2
-fail_under = 97.54
+fail_under = 98.79
 skip_covered = True
 show_missing = True
 exclude_lines =
     @abc.abstractmethod
+omit =
+    docs/src/logging/tutorial003.py
```

Comment:
Maybe just totally omit the docs/src directory?

Reply:
docs/src should be included in coverage to make sure that all code snippets from docs are tested. those tutorials are ommited because they display usage of `sentry`, `rollbar` and `raygun`, which cannot be tested